test/utils/bytecode.hpp

```diff
@@ -17,6 +18,8 @@ inline bytecode push(uint64_t n);
 inline bytecode push(evmc::address addr);
 inline bytecode push(evmc::bytes32 bs);
 
+using namespace evmone;
```

Comment:
```suggestion
using enum evmone::Opcode;
```

Reply:
This change will require to change `Opcode opcode` to `evmone::Opcode opcode` in `Opcode` type param declaration.